postem/postem-app/src/main/webapp/WEB-INF/templates/verify.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+   xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/common :: head" />
+<body>
+    <div th:replace="fragments/common :: jQuery" />
+    <div class="portletBody">
+          <div id="menu" th:include="fragments/menus :: main (index)" />
+          <h1 th:text="#{title_verify}"></h1>
+          <br>
```

Comment:
The page title (the h1) should be wrapped in a div with the .page-header class. Then you won't need the br line break.

Reply:
Added.